### PR TITLE
react evm sdk 

### PR DESCRIPTION
**Pull request by AlekseiPetrovDev**

### CODE REVIEW

**Review by zhelezkov — APPROVED** [review]
looks good

**Review by Kiryous — APPROVED** [review]
LGTM!

File: examples/bottom-chat-multichain/pages/index.tsx [hunk 1]
```diff
@@ -110,12 +119,31 @@ export default function Home(): JSX.Element {
...
     useState<DialectAptosWalletAdapter | null>(null);
 
+  const [dialectEvmWalletAdapter, setDialectEvmWalletAdapter] =
+    useState<DialectEvmWalletAdapter | null>(null);
+
+  useEffect(() => {
+    if (!evmWalletConnected || !evmWallet) {
+      setDialectEvmWalletAdapter(null);
+    } else {
+      console.log('ADSD');
```
Comment (zhelezkov) [inline examples/bottom-chat-multichain/pages/index.tsx:136]: remove console log

File: examples/inbox-multichain/pages/index.tsx [hunk 2]
```diff
@@ -69,11 +78,29 @@ export default function Home(): JSX.Element {
...
   const [dialectAptosWalletAdapter, setDialectAptosWalletAdapter] =
     useState<DialectAptosWalletAdapter | null>(null);
+  const [dialectEvmWalletAdapter, setDialectEvmWalletAdapter] =
+    useState<DialectEvmWalletAdapter | null>(null);
+
+  useEffect(() => {
+    if (!evmWalletConnected || !evmWallet) {
+      setDialectEvmWalletAdapter(null);
+    } else {
+      console.log('ADSD');
```
Comment (zhelezkov) [inline examples/inbox-multichain/pages/index.tsx:94]: remove console log

File: examples/inbox-multichain/pages/index.tsx [hunk 3]
```diff
@@ -69,11 +78,29 @@ export default function Home(): JSX.Element {
...
     useState<DialectSolanaWalletAdapter | null>(null);
   const [dialectAptosWalletAdapter, setDialectAptosWalletAdapter] =
     useState<DialectAptosWalletAdapter | null>(null);
+  const [dialectEvmWalletAdapter, setDialectEvmWalletAdapter] =
+    useState<DialectEvmWalletAdapter | null>(null);
+
+  useEffect(() => {
+    if (!evmWalletConnected || !evmWallet) {
+      setDialectEvmWalletAdapter(null);
+    } else {
```
Comment (zhelezkov) [inline examples/inbox-multichain/pages/index.tsx:93]: wouldn't it cause problems when other chains connected?

**Review by zhelezkov — APPROVED** [review]
looks good, minor comments

File: examples/inbox-multichain/pages/index.tsx (hunk 3 above)
Comment (AlekseiPetrovDev) [reply examples/inbox-multichain/pages/index.tsx:93]: no, it's shouldn't

**Pull request merged**